tests/bson/bson-regex-004.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+MongoDB\BSON\Regex debug handler
```

Comment:
Can we add "with flags omitted" as a suffix?

Alternatively, I have no qualms about adding this to `bson-regex-002.phpt`, which is the existing debug handler test. The test itself is basically a one-liner if we do away with the variable assignment.